tests/integrationtest/t/planner/core/integration_partition.test

```diff
@@ -541,6 +541,7 @@ select * from t2;
 select * from t2;
 drop table t2;
 drop database issue_27532;
+set autocommit = DEFAULT;
```

Comment:
Is this the reason for the plan changes, where it removes the projection and Union Scan? (I.e. no need to check for 'dirty' data in the current transaction)

Reply:
Yes. I think it is a mistake before.